provider/maas/environ.go

```diff
@@ -948,6 +970,18 @@ func (environ *maasEnviron) StartInstance(args environs.StartInstanceParams) (
 			return nil, errors.Annotatef(err, "cannot run instances")
 		}
 	}
+	done := make(chan struct{})
+
+	if args.StatusCallback != nil {
+		// the purpose of this goroutine is to provide information while the instance
+		// is coming up, once StartInstance exits instancepoller will take care but
+		// long start times can lead to confusing status stalling, there is no
```

Comment:
how slow is the code below this if block? isn't starting a node mostly asynchronous? I'm curious to know how much of a delay we're saving here